lib/hanami/model/configuration.rb

```diff
@@ -122,6 +121,32 @@ def logger=(value)
         return if value.nil?
         gateway.use_logger(@logger = value)
       end
+
+      def rom
+        @rom ||= ROM::Configuration.new(@backend, @url, infer_relations: false)
```

Comment:
From what I get `infer_relations` will default to `false` but later (`#load!`) that can get set to `true` if so desired.